_data/simulations/Spinodal_c/meta.yaml

```diff
@@ -0,0 +1,66 @@
+_id: 42f97720-a21c-11e9-9cca-0935b5963be4
+metadata:
+  author:
+    first: Shubhajit
+    last: Mondal
+    email: sm44@iitbbs.ac.in
+    github_id: smondal44
+  timestamp: '17 June, 2019'
+  summary: >-
+    Spinodal decomposition problem for T-shape domain has been solved using
+    open-source finite element software FeniCS.
+  implementation:
+    name: Fenics
+    repo:
+      url: >-
+        https://github.com/smondal44/spinodal-decomposition/blob/master/spinodal_c_conservative.py
+      version: b96a5defefc814178d294a1e5ec595cde5c2eb7f
+    container_url: ''
+  hardware:
+    cpu_architecture: x86_64
+    acc_architecture: non-gpu
+    parallel_model: serial
+    clock_rate: '2.4'
+    cores: '1'
+    nodes: '1'
+benchmark:
+  id: 1a
```

Comment:
This is benchmark problem 1c, with a T-shaped rather than square domain.

Reply:
@tkphd Yeah, I changed the id of the upload and uploaded again. Please check it.